src/vm/jitinterface.cpp

```diff
@@ -9953,8 +9953,8 @@ void* CEEInfo::getPInvokeUnmanagedTarget(CORINFO_METHOD_HANDLE method,
 
 /*********************************************************************/
     // return address of fixup area for late-bound N/Direct calls.
-void* CEEInfo::getAddressOfPInvokeFixup(CORINFO_METHOD_HANDLE method,
-                                        void **ppIndirection)
```

Comment:
> I think a better name for this method would be getAddressOfPInvokeTarget since the result may or may not be a fixup now.

@CarolEidt @BruceForstall @jkotas: I wonder if it would be worth combining `getAddressOfPInvokeFixup` and `getFunctionEntryPoint`. This may also allow us to share code between `Lowering::LowerNonvirtPinvokeCall` and `Lowering::LowerDirectCall`, which really only differ in that the former inserts a few P/Invoke-specific constructs around the call. If we were to do this, both `getAddressOfPInvokeFixup` and `getPInvokeUnmanagedTarget` would be dead code in CoreCLR and could be marked obsolete.